app/containers/message/Audio.js

```diff
@@ -225,6 +224,7 @@ class MessageAudio extends React.Component {
 			if (paused) {
 				await this.sound.pauseAsync();
 			} else {
+				await Audio.setAudioModeAsync(mode);
```

Comment:
Is this change needed?

Reply:
Yes, `setAudioModeAsync` needs to be called everytime we need to play audio